docs/report/report.tex

```diff
@@ -45,6 +45,105 @@ \subsubsection{Stretch Goals}
+Dafür kann die gleiche Library wie zum lesen von NFC-Tags genutzt werden.
+
+\subsection{Raspberri Pi}
+\subsubsection{Docker Integration}
+Auf einem Raspberri Pi Docker zu installieren und zum laufen zu bringen wird in vielen Anleitungen online beschrieben\footnote{https://phoenixnap.com/kb/docker-on-raspberry-pi}.
+
+\subsubsection{Öffentlich zugängliches Web Interface}
+Auf einem Raspberri Pi könnte eine Web Anwendung gehosted werden, welche die allgemeine Web Anwendung für alle Boxen darstellen soll.
+Um diese Web Anwendung von außerhalb des eigenen Netzwerkes erreichen zu können, muss innerhalb des Routers der Port 80 ge-forwarded werden.
+Dann kann der Pi unter dessen öffentlicher IP xxx.xxx.xxx.xxx:80 erreicht werden.
+Da sich die öffentliche IP-Adresse eines privaten Internet-Anschlusses in der Regel täglich ändert, wird zum einfachen finden der IP ein DynDns Service benötig, welcher eine feste Domain in die wechselnde IP Adresse des Routers übersetzt.
+Alternativ ginge es auch ohne Port-Forwarding mit nginx und ngrok \footnote{https://vatsalyagoel.com/setting-up-a-public-web-server-using-a-raspberry-pi-3/}.
+Für Unerfahrene sind das zu Beginn etwas komplexere Thematiken, effektiver Arbeitsaufwand hängt daher auch sehr stark von der Erfahrung der einzelnen Teammitglieder ab.
+
+\subsubsection{URL für UI festlegen}
+Über ein DNS Service, der auf dem Raspberri Pi läuft, wäre es möglich für die statische Public-IP eine eigene URL anzulegen.
+Dafür sind verschiedene DNS Services möglich, potentiell ist auch eine Domain notwendig.
+
+\subsection{User Interface}
+\subsubsection{Zugriff auf NFC Reader von Cloud Anwendung}
+Um von der Cloud Anwendung auf die Daten vom NFC Reader der verschiedenen Boxen zuzugreifen, wäre eine API sinnvoll.
+Der Chip der Box könnte beim auslesen von neuen NFC-Tag Daten einen API Call an die Cloud Anwendung schicken, welche das jeweilige hinterlegte Event zu triggern.
+
+\subsubsection{Login via Spotify, Youtube, etc.}
+Es gibt ein Feathers Plugin, welches die Möglichkeit bietet OAuth Provider zu nutzen, um sich über andere Services wie Spotify anzumelden.
+
+\subsubsection{Gleichen Nutzer bei verschiedenen Loginvarianten wiedererkennen}
+Um gleiche Nutzer zu erkennen, müssten Merkmale angelegt werden, über die diese Nutzer wiedererkennbar wären.
+Die E-Mail wäre hierbei ein geeignetes Merkmal, da dies einzigartig ist. Über gesetzte Scopes in der OAuth Anfrage kann diese vom jeweiligen Provider mitgeliefert werden.
+Um die E-Mail als Wiedererkennungsmerkmal zu verwenden, muss vorausgesetzt sein, dass Nutzer immer die gleiche E-Mail bei den unterschiedlichen Providern nutzen. Dies ist aber nicht immer die Regel.
+Daher könnte dem (bereits eingeloggten) Nutzer die Möglichkeit gegeben werden, weitere Accounts zu dem bestehenden hinzuzufügen.
+Dabei könnte dann der neue Account zu dem bereits bestehendem Account hinzugefügt und entsprechend in der Datenbank hinterlegt werden.
```

Comment:
Die beiden Sätze klingen sehr ähnlich und können in einen verständlicheren einzelnen Satz umformuliert werden.

Reply:
added suggestion in commit 918a035f2290a2f9a2d45e7538efde98790a9fe1